source/configuration_manual/fts/tokenization.rst

```diff
@@ -96,24 +96,30 @@ Base64 detection
 .. versionadded:: v2.3.18
 
 Base64 sequences are looked for in the tokenization buffer and skipped when detected.
-The detection works with what is already present in the buffer and it does not attempt to pull further data to complete a match.
 
-A base64 sequence is composed by:
+A base64 sequence is detected by:
 
-  * an optional leader character comprised in ``leader-characters`` set.
-  * a run characters all comprised in the ``base64-characters`` set, at least ``minimum-run-length`` long.
-  * an end-of-buffer or a trailer character comprised in ``trailer-characters`` set,
+  * an optional leader character comprised in ``leader-characters`` set,
+  * a run of characters, all comprised in the ``base64-characters`` set, at least ``minimum-run-length`` long,
+  * an end-of-buffer, or a trailer character comprised in ``trailer-characters`` set,
 
 where:
 
-  * leader-characters: ``[ \t\r\n=:;?]``
-  * base64-characters: ``[0-9A-Za-z/+]``
-  * trailer-characters: ``[ \t\r\n=:;?]``
-  * minimum run length: ``50``
-  * minimum runs count: ``1``
+  * ``leader-characters`` are: ``[ \t\r\n=:;?]``
+  * ``base64-characters`` are: ``[0-9A-Za-z/+]``
+  * ``trailer-characters`` are: ``[ \t\r\n=:;?]``
+  * ``minimum-run-length`` is: ``50``
+  * ``minimum-run-count`` is: ``1``
 
-i.e. (even single) 50-chars runs of characters in the base64 set are recognized as base64 and ignored in indexing.
+e.g. (even single) 50-chars runs of characters in the base64 set are recognized as
+base64 and ignored in indexing.
+
+Look ahead for the entire base64 run is not performed. Instead we match the sequences
+as the text chunks are pulled into the tokenization buffer.
+Given the rules above, the indexed extra content, due to sequences split across chunks,
+does not impact significantly the quality of the filter.
```

Comment:
This is a bit difficult to read and not entirely clear what it means. Maybe:

If the base64 sequences happen to be split across different chunks, they might not be detected as base64. In this case the base64 content is still indexed. However, this happens rarely enough that it does not significantly impact the quality of the filter.